airflow/configuration.py

```diff
@@ -372,6 +375,9 @@ def run_command(command):
 # default_principal = admin
 # default_secret = admin
 
+[admin]
+hide_confidential_ui_fields = true
```

Comment:
Should be `True` just to be more Python-esque, e.g.

     # statsd_on =  False